app/RangeTest/README.md

```diff
@@ -0,0 +1,14 @@
+# RadioTest
+
+#### set up the mote
```

Comment:
Thanks for the explanation. I'm happy to test things out if you don't have the setup in front of you.